docs/guides/usage-with-react.md

```diff
@@ -107,8 +107,7 @@ class BooksList extends Component {
       return;
     }
 
-    const nextSearchStatus = getStatus(nextProps, 'books.requests.getSearchResults.status');
-
+    const nextSearchStatus = nextProps.searchStatus;
```

Comment:
@ianmstew I know you said you'd grab this one but I just had to make the change while I was in the docs source 🙈